docs/Reference/Hooks.md

```diff
@@ -267,6 +268,26 @@ service (if the `connectionTimeout` property is set on the Fastify instance).
 The `onTimeout` hook is executed when a request is timed out and the HTTP socket
 has been hanged up. Therefore, you will not be able to send data to the client.
 
+### onClientAbort
+
+```js
+fastify.addHook('onClientAbort', (request, reply, done) => {
+  // Some code
+  done()
+})
+```
+Or `async/await`:
+```js
+fastify.addHook('onClientAbort', async (request, reply) => {
+  // Some code
+  await asyncMethod()
+})
+```
+`onClientAbort` is useful if you need to monitor the if the client aborts the request
```

Comment:
Can you add a disclaimer like https://github.com/fastify/fastify/blob/main/docs/Guides/Detecting-When-Clients-Abort.md#conclusion
It would never be a reliable way to detect request abort, but works only the other side properly close the socket.

Reply:
Done!